controller.js

```diff
@@ -1,14 +1,14 @@
-const { topics } = require("./model");
-const endpoint =require("./endpoints.json") 
+// const { topics } = require("./Models/getTopics.model");
```

Comment:
do remove unused/commented out code or files that are no longer needed before pushing